snooty/test_parser.py

```diff
@@ -1971,66 +1971,67 @@ def test_footnote_reference() -> None:
     )
 
 
-def test_cardgroup() -> None:
-    path = ROOT_PATH.joinpath(Path("test.rst"))
-    project_config = ProjectConfig(ROOT_PATH, "", source="./")
-    parser = rstparser.Parser(project_config, JSONVisitor)
-
-    page, diagnostics = parse_rst(
-        parser,
-        path,
-        """
-.. card-group::
-   :type: small
-
-    cards:
-
-    - id: test-one
-      headline: Test One
-      image: /compass-explain-plan-with-index-raw-json.png
-      link: https://docs.mongodb.com/guides/server/delete
-
-    - id: test-two
-      headline: Test Two
-      image: /compass-explain-plan-with-index-raw-json.png
-      link: https://docs.mongodb.com/guides/server/update
-""",
-    )
-    page.finish(diagnostics)
-    assert len(diagnostics) == 0
-    check_ast_testing_string(
-        page.ast,
-        """<root fileid="test.rst">
-        <directive name="card-group" type="small">
-        <directive name="cardgroup-card" cardid="test-one" headline="Test One" image="/compass-explain-plan-with-index-raw-json.png" link="https://docs.mongodb.com/guides/server/delete" checksum="e8d907020488a0b0ba070ae3eeb86aae2713a61cc5bb28346c023cb505cced3c" />
-        <directive name="cardgroup-card" cardid="test-two" headline="Test Two" image="/compass-explain-plan-with-index-raw-json.png" link="https://docs.mongodb.com/guides/server/update" checksum="e8d907020488a0b0ba070ae3eeb86aae2713a61cc5bb28346c023cb505cced3c" />
-        </directive>
-        </root>""",
-    )
-
-
-def test_cardgroup_invalid_yaml() -> None:
-    path = ROOT_PATH.joinpath(Path("test.rst"))
-    project_config = ProjectConfig(ROOT_PATH, "", source="./")
-    parser = rstparser.Parser(project_config, JSONVisitor)
-    page, diagnostics = parse_rst(
-        parser,
-        path,
-        """
-.. card-group::
-
-    cards:
-
-    - id test-one
-      headline: Test One
-      image: /compass-explain-plan-with-index-raw-json.png
-      link: https://docs.mongodb.com/guides/server/delete
-""",
-    )
-    page.finish(diagnostics)
-    assert len(diagnostics) == 1
-    assert isinstance(diagnostics[0], ErrorParsingYAMLFile)
-    assert diagnostics[0].start[0] == 5
+# DEPRECATING?
```

Comment:
This content format is used in the BI connector docs on the [Connect Client Applications page](https://docs.mongodb.com/bi-connector/current/client-applications/) ([source](https://github.com/mongodb/docs-bi-connector/blob/master/source/client-applications.txt#L32-L75)). I'm fine deprecating this if we provide them with an alternative and send in a PR with any relevant content changes to use the alternative solution.